### ISSUE TITLE
CWL conformance tests on ci.commonwl.org have been completely failing for a while

### ISSUE THREAD

**Issue by mr-c** [broadinstitute/cromwell#3973]
Example: https://ci.commonwl.org/job/cromwell/230/console

**Comment by mr-c** [comment]
Yes, now that tests have IDs we will only append and not insert new tests. Thanks for the update!

**Comment by geoffjentry** [comment]
Hey @mr-c - I'm looking to update things.

But this raises another question - since CWL 1.0.X is by definition backwards compatible, why would our reliance on an older version of `schema-salad` be an error on the tester? It shouldn't matter? 

For instance we've talked in the past about embedding a particular version of the python code in our JAR just to ensure stability, but if I'm understanding what's causing issues on the CI (and I'm not sure I **am**, but it's my current theory at least) then if we did such a thing this would come up regardless.
